pom.xml

```diff
@@ -93,8 +86,13 @@
 			<groupId>org.hamcrest</groupId>
 			<artifactId>hamcrest</artifactId>
 		</dependency>
+        <dependency>
```

Comment:
Is this dependency used? If it is, for what purpose?

Reply:
No, I'm not using it 
When I started this project I took this pom from tutorial repo 
Thank you for reviewing